cfme/services/service_catalogs/__init__.py

```diff
@@ -99,6 +99,6 @@ def fill(self, fill_data):
         return was_change
 
 
-from . import ui, ssui  # NOQA last for import cycles
+from cfme.services.service_catalogs import ui, ssui  # NOQA last for import cycles
```

Comment:
Not separating these imports?

Reply:
Leaving them with `# NOQA` as a 1-line import for importscan.